src/Pages/Main/MainBanner/MainBanner.component.js

```diff
@@ -0,0 +1,161 @@
+import React, { Component } from "react";
+import { Link } from "react-router-dom";
+
+import "./MainBanner.styles.scss";
+
+class MainBanner extends Component {
+  constructor() {
+    super();
+    this.state = {
+      bannerImages: [],
+      bannerImgWidth: window.innerWidth,
+      bannerImgCount: 0,
+      currentBannerId: 1,
+      bannerImgsXcoordinate: 0,
+      bannerImgsTransitionDelay: 800,
+      leftClickedOnMainBanner: false,
+      slideIntervalDelay: 4000,
+    };
+  }
+
+  componentDidMount = () => {
+    const { slideIntervalDelay } = this.state;
+    fetch("http://localhost:3000/data/main/MainBannerImagesData.json")
```

Comment:
```suggestion
    fetch("/data/main/MainBannerImagesData.json")
```
포트번호를 정의해주지 않아도 되기 때문에 이렇게 쓰는게 더 좋은 것 같네요.

Reply:
오호 이런방법이... 적용했습니다~~